stix_shifter_modules/secretserver/README.md

```diff
@@ -99,3 +118,9 @@ python3 main.py execute secretserver secretserver '{"type": "identity", "id": "i
 }
 
 ```
+Supported queries for secret server conncetor:
```

Comment:
You should say these are examples of supported queries since more operators than just `LIKE` are supported.

Reply:
Done the changes.